Create CONTRIBUTING.md

hoverduck opened the pull request.

koke commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,19 @@
+
+## Submitting code changes
+
+We welcome Pull Requests for any new feature or bug fix that you'd like to contribute!  More details on our process can be found in the [Make WordPress Mobile handbook](https://make.wordpress.org/mobile/handbook/pathways/ios/), specifically [here](https://make.wordpress.org/mobile/handbook/pathways/ios/how-to-contribute/).
```
We definitely welcome PRs for bug fixes, but I'm a bit concerned about encouraging PRs for new features without previous discussion. I wouldn't want someone to spend a few days working on something we might not want to include in the app.

catehstn replied:
Yeah I would have to agree with this. Can we put something about if it's an untracked issue, they should first open an issue with some mention of how they would like to approach it?

Related:
- What about the .org slack? Are people in there? Is that somewhere we can be more active?
- Should we be more open about our roadmap? What would that look like?

hoverduck commented:
Good points @koke / @catehstn - I've removed the note about PRs for feature requests and added a note about .org Slack (taken from https://make.wordpress.org/mobile/handbook/communication/dev-chats/)
